src/Integration/RequestFetcher.php

```diff
@@ -22,6 +22,6 @@ public function fetchRequest(): ?ServerRequestInterface
             return null;
         }
 
-        return ServerRequest::fromGlobals();
+        return (new WellKnownPsr17Factory())->createServerRequestFromGlobals();
```

Comment:
This removes the "require" coupling to "guzzlehttp/psr7": "well-known-implem" provides the same feature, but compatible with any PSR-7 implementations.